File: src/co/soundwave/repository/album/AlbumSQL.kt
package co.soundwave.repository.album

import co.soundwave.repository.BaseSQL

class AlbumSQL : BaseSQL() {
    fun insertAlbum(name: String, path: String, year: Int, artistId: Int): String =
        "insert into ${tableName()}(name_${tableName()}, img_path_${tableName()}," +
            " release_year_${tableName()}, artist_id_artist) values ('$name', '$path', '$year','$artistId');"
    
    override fun tableName(): String = "album"
}

File: repository/album/AlbumSQL.py
from repository import BaseSQL as bq


class AlbumSQL(bq.BaseSQL):

    def insert_album(self,_id,_name,_img_path_album ,_release_year_album,_artist_id_artist):

        return "insert into %s values(%s,%s,%s,%s,%s);" % (
            self.table_name(), _id, _name, _img_path_album, _release_year_album, _artist_id_artist)

    def table_name(self):
        return "album"

    def abc(self):
        pass
